common/oatbox/TaskQueue/AbstractTask.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace oat\oatbox\TaskQueue;
+
+/**
+ * Class AbstractTask
+ *
+ * @author Gyula Szucs <gyula@taotesting.com>
+ */
+abstract class AbstractTask extends Message implements TaskInterface
+{
+    private $parameters = [];
```

Comment:
Should it be private? Classes which will extend `AbstractTask` will be not able to work with this property.